### PR TITLE
Code cleanups suggested by gosimple

### PR DESCRIPTION
**Pull request by dgryski**

### CODE REVIEW

File: platforms/gpio/motor_driver.go [hunk 1]
```diff
@@ -175,10 +175,7 @@ func (m *MotorDriver) Direction(direction string) (err error) {
 }
 
 func (m *MotorDriver) isDigital() bool {
-	if m.CurrentMode == "digital" {
-		return true
-	}
-	return false
+	return m.CurrentMode == "digital"
```
Comment (trevrosen) [inline platforms/gpio/motor_driver.go]: I like capturing this kind of thing in a constant.

**Comment by trevrosen** [comment]
:shipit: when 🍏

**Comment by dgryski** [comment]
I have no idea why the coverage dropped, nor why the test failed :(

**Comment by trevrosen** [comment]
Looks like this test broke, potentially b/c of some kind of ephemeral clock value: https://travis-ci.org/hybridgroup/gobot/jobs/144491653#L727

@deadprogram I'm thinking that [this delay might not be sufficient](https://github.com/hybridgroup/gobot/commit/3cf667d4507c073e57ccce47746695023ffabc06#diff-c6ba88cd6918ca83532e2a61dba656e8R31) for the infrastructure Travis is running it on. Just a guess tho, haven't really delved any deeper.

**Comment by deadprogram** [comment]
@trevrosen I think you are correct, but even a larger value seems to have test failures occasionally. I'd love some help looking into it, or suggestions.

@dgryski as you said, your changes certainly did not impact the test. I restarted the build, and all passed. Hence, I am merging.

**Pull request merged**